python/feldera/formats.py

```diff
@@ -0,0 +1,83 @@
+from typing import Optional
+from typing_extensions import Self
+from enum import Enum
+
+
+class UpdateFormat(Enum):
+    """
+    Enum for the update format of the JSON format.
+
+    https://www.feldera.com/docs/api/json/#the-insertdelete-format
+    """
+
+    InsertDelete = 1
+    """
+    This is a way to represented changes in the data.
+    When used, the actual JSON data is expected to be inside one of the keys "insert" or "delete".
```

Comment:
I think that the example is clearer than this text, which can be removed.

Reply:
just explain that 'id' and 'name' below are the columns of the table where the insertion and deletion happen.